src/server.ts

```diff
@@ -38,8 +40,7 @@ app.use(passport.initialize());
 //Use body-parser middleware
 app.use(bodyParser.urlencoded({ extended: false }));
 app.use(bodyParser.json());
-app.use('/api', routes);
-app.use('/api', changePasswordIgnored, routes);
+app.use('/api', routes, authenticate, wishlistRoutes, changePasswordIgnored);
```

Comment:
This is not global middleware, it has to be imported in with wishlist routes

Reply:
no the reason i made this global is because i wanted it to be used by anyone who want to check if a user is logged in